src/main/groovy/edu/oregonstate/mist/invencycle/resources/BikeResource.groovy

```diff
@@ -0,0 +1,56 @@
+package edu.oregonstate.mist.invencycle.resources
+
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.invencycle.core.Bike
+import edu.oregonstate.mist.invencycle.core.Sample
+import edu.oregonstate.mist.api.AuthenticatedUser
+import edu.oregonstate.mist.invencycle.db.BikeDAO
+import edu.oregonstate.mist.invencycle.core.ErrorPOJO
+import io.dropwizard.auth.Auth
+import io.dropwizard.jersey.params.IntParam
+import javax.ws.rs.GET
+import javax.ws.rs.POST
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.Consumes
+import javax.ws.rs.core.Response
+import javax.ws.rs.core.Response.ResponseBuilder
+import javax.ws.rs.core.MediaType
+
+/*
```

Comment:
This is a valid java comment, but it's not javadoc. What you want to use here is the javadoc standard: 
`/**
 * Foo Bar
 */`

See: https://en.wikipedia.org/wiki/Javadoc